daras_ai_v2/bots.py

```diff
@@ -133,15 +131,8 @@ def __init__(self):
         elif should_translate_lang(user_language):
             self.user_language = user_language
 
-        if self.bi.workspace:
-            self.workspace = self.bi.workspace
-            self.current_user = self.bi.created_by
-        else:
-            # TODO: remove this once all bots have been migrated to workspaces
-            self.workspace = Workspace.objects.get_or_create_from_uid(
-                self.bi.billing_account_uid
-            )[0]
-            self.current_user = self.workspace.created_by
+        self.workspace = self.bi.workspace
+        self.current_user = self.bi.created_by
```

Comment:
no longer needed.

i confirmed that there are no bot integrations that do not have a workspace